### PR TITLE
Fix crashing errors in iOS

### PR DESCRIPTION
**Pull request by jeffswartz**
### Contributing checklist
- [ ] Code must follow existing styling conventions
- [x] Added a descriptive commit message
- [ ] Sample apps updated if needed

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# 2.27.6  (June 2024)
+
+- [Fix]: This version fixes some iOS crashes that were introduced in version 2.275
```
Comment (marinaserranomontes) [inline CHANGELOG.md:3]: Typo error: it should be 2.27.5

Comment (jeffswartz) [inline CHANGELOG.md:3]:
```suggestion
- [Fix]: This version fixes some iOS crashes that were introduced in version 2.27.5.
```

**Review by marinaserranomontes — APPROVED** [review]
LGTM

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+# 2.27.6  (June 2024)
+
+- [Fix]: This version fixes some iOS crashes that were introduced in version 2.27.5.
```
Comment (jeffswartz) [inline CHANGELOG.md:3]:
```suggestion
- [Fix]: This version fixes some iOS crashes that were introduced in version 2.27.5. Fixes issue #757.
```

**Pull request merged**